### PR TITLE
Implemented rtl layout support

### PR DESCRIPTION
**Pull request by saeedrahimi**
Hi there.
I've added rtl layout support to this component.
changes the tab direction with direction='rtl' property

- Added rtl demo
- updated readme

### CODE REVIEW

**Comment by saeedrahimi** [comment]
I have no idea why CI fails.

**Comment by zombieJ** [comment]
It's eslint latest version bug. Could you merge master, I add an now script to preview the effect.

File: README.md [hunk 1]
```diff
@@ -375,6 +393,12 @@ swipeable tab bar with ink indicator, same with tab bar/ink bar props, and below
           <td>false</td>
           <td>whether animate tabpane with css margin</td>
       </tr>
+      <tr>
+          <td>direction</td>
```
Comment (zombieJ) [inline README.md]: No need to export `direction` api everywhere. It's ok on Tabs & TabBar.

File: src/InkTabBarNode.js [hunk 2]
```diff
@@ -44,7 +44,9 @@ function componentDidUpdate(component, init) {
           left += (tabNode.offsetWidth - width) / 2;
         }
       }
-
+      if (direction === 'rtl') {
+        left = (left * -1) + getStyle(tabNode, 'margin-left');
```
Comment (zombieJ) [inline src/InkTabBarNode.js]: `getStyle(tabNode, 'margin-left') - left`?

File: src/ScrollableTabBarNode.js [hunk 3]
```diff
@@ -141,15 +141,18 @@ export default class ScrollableTabBarNode extends React.Component {
...
-          navOffset = {
-            value: `translate3d(${target}px,0,0)`,
-          };
-        } else {
-          navOffset = {
-            name: 'left',
-            value: `${target}px`,
-          };
+        if (this.props.direction === 'rtl') {
+          target = target * -1;
```
Comment (zombieJ) [inline src/ScrollableTabBarNode.js]: `target = -target`

**Comment by zombieJ** [comment]
So cool! I'm OK with this.

**Comment by saeedrahimi** [comment]
@zombieJ 
I resolved issues you've mentioned in conversation, but there is a problem.
i made a mistake last days and deleted tabs fork from my github, hopefully i restored my changes and pushed it again to [https://github.com/saeedrahimi/tabs/tree/support-rtl](https://github.com/saeedrahimi/tabs/tree/support-rtl).
the problem is this pull request that referring to unknown-repository.
Do you have any idea to help me on this? please

**Comment by zombieJ** [comment]
You can checkout commit by this pr commit hash and create a new repo. Github will refresh info with the PR.

**Comment by saeedrahimi** [comment]
> You can checkout commit by this pr commit hash and create a new repo. Github will refresh info with the PR.

i can get pull request changes by running this command
```
git fetch https://github.com/react-component/tabs pull/227/head:support-rtl
```
but when i push to a new repo, `unknown-repository` doesn't change.

According to this [issue](https://github.com/isaacs/github/issues/168) i should open a new pull request.

**Pull request closed without merging**